### PR TITLE
GEOMESA-1375 Rewriting stats parsing in parboiled

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/BasicParser.scala [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+package org.locationtech.geomesa.utils.text
+
+import java.util.regex.Pattern
+
+import org.apache.commons.lang.StringEscapeUtils
+import org.parboiled.scala._
+
+/**
+  * Base class por parboiled parsers that provides methods for string and number matching
```
Comment (jnh5y) [inline geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/BasicParser.scala]: ^por^for?

Comment (elahrvivaz) [inline geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/BasicParser.scala]: por favor

Comment (jnh5y) [inline geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/BasicParser.scala]: Poor flavor wha?  

No, I was thinking that "Base class por parboiled parsers" might should be "Base class for parboiled parsers".

I'm fine if we internationalize the docs.:)  @mdzimmerman what's an ETA on Polish?

Comment (elahrvivaz) [inline geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/BasicParser.scala]: :)
fixed

**Pull request merged**